assets/activation-scripts/activate.d/start-services.bash

```diff
@@ -18,23 +20,31 @@ poll_services_status() {
 }
 
 wait_for_services_socket() {
+  local _sleep="@coreutils@/bin/sleep"
   local socket_file="$1"
   local status
   status=$(poll_services_status "$socket_file")
   while [ "$status" == "$NOT_READY" ]; do
-    "$_coreutils/bin/sleep" 0.02
+    "$_sleep" 0.02
     status=$(poll_services_status "$socket_file")
   done
 }
 
 start_services_blocking() {
+  local _bash="@bash@/bin/bash"
+  local _cat="@coreutils@/bin/cat"
+  local _date="@coreutils@/bin/date"
+  local _jq="@jq@/bin/jq"
+  local _process_compose="@process_compose@/bin/process-compose"
+  local _setsid="@util_linux@/bin/setsid"
+  local _timeout="@coreutils@/bin/timeout"
```

Comment:
why dont we declare imports at top of the file here as we do for the actier activate scripts?

Reply:
That's what I did at first but then those declarations didn't appear within the functions in that script! I was a bit surprised by this / had forgotten this detail of bash variable scoping as well. Turns out this is the only file with functions so that's why we don't see the same issue elsewhere.